docassemble/ALToolbox/data/questions/al_income_demo.yml

```diff
@@ -403,11 +410,11 @@ content: |
   
   Name | Calculation type | Frequency | Gross amount
   - | - | - | -
-  % for name, item in x.to_add.complete_elements().items():
-  ${ job_items_names.get(name, name) } | Gross | Monthly | ${ currency(x.gross_total( source=name, times_per_year=12 )) }
+  % for source, item in x.to_add.complete_elements().items():
+  ${ job_items_names.get(source, source) } | Gross | Monthly | ${ currency(x.gross_total( source=source, times_per_year=12 )) }
   % endfor
   % for name, item in x.to_subtract.complete_elements().items():
```

Comment:
I think this last for loop is using `source` again instead of `name`. When I went through the interview, I had two items to subtract.
```suggestion
  % for source, item in x.to_subtract.complete_elements().items():
```

I had two deductions and I got two duplicate rows, both showing `to_add` at the final screen:

<img width="477" alt="dup_rows" src="https://user-images.githubusercontent.com/52798256/216471493-9fb88a9a-d2aa-4df9-9bb8-43fd1d601153.png">